flare_portal/experiments/templates/experiments/participant_list.html

```diff
@@ -79,7 +79,7 @@ <h3 class="card-title">Participants</h3>
                                             </td>
 
                                             <td>
-                                                <span class="status-icon {% if not form.instance.has_been_rejected %}bg-success{% else %}bg-danger{% endif %}"></span> {{ form.instance.has_been_rejected|yesno|title }}
+                                                <span class="status-icon {% if form.instance.has_been_rejected %}bg-success{% else %}bg-danger{% endif %}"></span> {{ form.instance.has_been_rejected|yesno|title }}
```

Comment:
I don’t see why dropping the `not` is appropriate here?

Reply:
The client wanted to reverse the colours used in the table :)